YellowNotes.Api/YellowNotes.Api.csproj

```diff
@@ -1,13 +1,24 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
+  <ItemGroup>
+    <ProjectReference Include="..\YellowNotes.Core\YellowNotes.Core.csproj"/>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.4.0"/>
+    <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.5"/>
+    <PackageReference Include="Microsoft.Extensions.Configuration.AzureKeyVault" Version="3.1.2"/>
+  </ItemGroup>
+  
+  <ItemGroup>
+    <Folder Include="Controllers\" />
+  </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\YellowNotes.Core\YellowNotes.Core.csproj" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="3.1.2" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.6.0" />
```

Comment:
hmmm.. I've never seen that packages are in a separate `ItemGroup`. would be nicer to have it together